setup.py

```diff
@@ -217,7 +217,8 @@ def setup_package():
                 'cuda90': ['cupy-cuda90>=4.0', 'thinc_gpu_ops>=0.0.3,<0.1.0'],
                 'cuda91': ['cupy-cuda91>=4.0', 'thinc_gpu_ops>=0.0.3,<0.1.0'],
                 'cuda92': ['cupy-cuda92>=4.0', 'thinc_gpu_ops>=0.0.3,<0.1.0'],
-                'cuda100': ['cupy-cuda100>=4.0', 'thinc_gpu_ops>=0.0.3,<0.1.0']
+                'cuda100': ['cupy-cuda100>=4.0', 'thinc_gpu_ops>=0.0.3,<0.1.0'],
+                'mecab': ['mecab-python3==0.7']
```

Comment:
Looks good 👍

Maybe we could call the extra `ja`, after the language shortcode? This would also allow us to put more dependencies in there if needed (and add all other languages that depend on external packages in a similar way).

```suggestion
                'ja': ['mecab-python3==0.7']
```

Reply:
Sounds good to me!